src/main/java/net/phptravels/gui/utils/TakeScreenshot.java

```diff
@@ -0,0 +1,46 @@
+package net.phptravels.gui.utils;
+
+import com.qaprosoft.carina.core.foundation.utils.R;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.logging.log4j.LogManager;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.logging.Logger;
+
+import static com.zebrunner.agent.core.webdriver.RemoteWebDriverFactory.getDriver;
+
+public class TakeScreenshot {
```

Comment:
implement IDriverPool

Reply:
Also it's utils class, please rename it to `TakeScreenshotUtils` or `TakeScreenshotService` - this names are more clear